pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -440,6 +440,51 @@ def remove_cpds(self, *cpds):
                 cpd = self.get_cpds(cpd)
             self.cpds.remove(cpd)
 
+    def get_cardinality(self, node=None):
+        """
+        Returns the cardinality of the node.
```

Comment:
I think we should also mention here that it returns all the cardinalities if `node=None`.